pkgs/tools/security/keybase-bin/default.nix

```diff
@@ -0,0 +1,66 @@
+with import <nixpkgs> { };
+
+let
+  libPath = stdenv.lib.makeLibraryPath [
+    alsaLib
+    atk
+    cairo
+    cups
+    dbus
+    expat
+    fontconfig
+    freetype
+    gcc.cc
+    gdk_pixbuf
+    glib
+    gnome2.GConf
+    gtk2
+    nspr
+    nss
+    pango
+    systemd
+    xorg.libX11
+    xorg.libxcb
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXScrnSaver
+    xorg.libXtst
+  ];
+in
+  stdenv.mkDerivation rec {
+    name = "keybase-bin-${version}";
+    # version = "1.0.25-20170714172717.73f9070";
+    version = with builtins; replaceStrings ["+"] ["."] (fromJSON (readFile (fetchurl "http://prerelease.keybase.io.s3.amazonaws.com/update-linux-prod.json"))).version;
+    src = builtins.fetchurl "https://s3.amazonaws.com/prerelease.keybase.io/linux_binaries/deb/keybase_${version}_amd64.deb";
```

Comment:
I thought you mentioned in IRC that you would only use this privately? I'm pretty sure this is not fit for nixpkgs, because it's not reproducible: Building this package now will yield a completely different than building it (at the exact same nixpkgs revision) in 5 years time. That's the reason there's hashes all over nixpkgs, because hashes guarantee that the content is exactly the same and therefore reproducible. If you want to provide an always up-to-date package, then you might want to create a separate repository as a [nixpkgs overlay](https://nixos.org/nixpkgs/manual/#chap-overlays), mozilla is [doing this with firefox, rust and more](https://github.com/mozilla/nixpkgs-mozilla).